dev/fattest.simplicity/src/componenttest/rules/repeater/JakartaEE11Action.java

```diff
@@ -69,7 +69,7 @@ public class JakartaEE11Action extends JakartaEEAction {
                                                   "cdi-4.1",
                                                   "concurrent-3.1",
                                                   "connectors-2.1",
-                                                  "data-1.0",
+                                                  "data-1.1",
```

Comment:
this could be an issue depending on how this gets used. jakartaee-11 and webprofile-11 bring in data-1.0. if these features are used in combination with each other that would throw an error. If the features are just used individually though, this is fine and you can ignore this comment